databricks/koalas/frame.py

```diff
@@ -2554,9 +2554,8 @@ def describe(self, percentiles=None) -> 'DataFrame':
 
         Parameters
         ----------
-        percentiles : list of ``float`` in range (0.0, 1.0), default [0.25, 0.5, 0.75]
+        percentiles : list of ``float`` in range [0.0, 1.0], default None
```

Comment:
I think we can say the default is [0.25, 0.5, 0.75] here.

Reply:
Done